to_data_library/data/transfer.py

```diff
@@ -335,7 +335,7 @@ def s3_to_bq(self, aws_session, s3_connection_string, bucket_name, object_name,
         Exports S3 file to BigQuery table
 
         Args:
-          aws_session: authenticated AWS session.
+                aws_session: authenticated AWS session.
           s3_connection_string (str): The S3 connection string in the format
                                       {region}:{access_key}:{secret_key}
```

Comment:
I don't believe we need to be talking about access_keys and secret_keys anymore? we're just passing around an aws_session

Reply:
Ive requested to remove already @kthhrv :)